go.mod

```diff
@@ -14,3 +14,67 @@ require (
 	k8s.io/klog v0.4.0
 	k8s.io/utils v0.0.0-20201110183641-67b214c5f920
 )
+
+require (
+	github.com/Microsoft/go-winio v0.4.16 // indirect
+	github.com/ProtonMail/go-crypto v0.0.0-20210428141323-04723f9f07d7 // indirect
+	github.com/acomagu/bufpipe v1.0.3 // indirect
+	github.com/beorn7/perks v1.0.1 // indirect
```

Comment:
we can combine the 2 `require` blocks...

Reply:
Running `make revendor` or `go mod vendor` will separate them again. That's how go v1.17 is designed to treat the indirect dependencies. Read more [here](https://golang.org/doc/go1.17#go-command)